lms/static/js/bookmarks/views/bookmark_button.js

```diff
@@ -1,6 +1,6 @@
 ;(function (define, undefined) {
     'use strict';
-    define(['gettext', 'jquery', 'underscore', 'backbone', 'js/views/message_banner'],
+    define(['gettext', 'jquery', 'underscore', 'backbone', 'lmsCoursewareMessageBanner'],
         function (gettext, $, _, Backbone, MessageBannerView) {
```

Comment:
The Jasmine tests are failing because you need to register ``lmsCoursewareMessageBanner`` in the spec main.js file:

https://github.com/edx/edx-platform/blob/master/lms/static/js/spec/main.js